dotcom-rendering/src/layouts/CrosswordLayout.tsx

```diff
@@ -0,0 +1,785 @@
+import { BannerWrapper, Stuck } from './lib/stickiness';
+
+const CrosswordGrid = ({ children }: { children: React.ReactNode }) => (
+	<div
+		css={css`
+			/* IE Fallback */
+			display: flex;
+			flex-direction: column;
+			${until.leftCol} {
+				margin-left: 0px;
+			}
+			${from.leftCol} {
+				margin-left: 151px;
+			}
+			${from.wide} {
+				margin-left: 230px;
+			}
+
+			@supports (display: grid) {
+				display: grid;
+				width: 100%;
+				margin-left: 0;
+				grid-column-gap: 20px;
+
+				/*
+					Explanation of each unit of grid-template-columns
+
+					Left Column
+					Main content
+					Right Column
+				*/
+				${from.wide} {
+					grid-template-columns: 220px 1fr 300px;
+
+					grid-template-areas:
+						'title  headline      right-column'
+						'meta   standfirst    right-column'
+						'meta   instructions  right-column'
+						'body   body          right-column';
+				}
+
+				/*
+					Explanation of each unit of grid-template-columns
+
+					Left Column
+					Main content
+					Right Column
+				*/
+				${until.wide} {
+					grid-template-columns: 140px 1fr 300px;
+
+					grid-template-areas:
+						'title  headline      right-column'
+						'meta   standfirst    right-column'
+						'meta   instructions  right-column'
+						'body   body          right-column';
+				}
+
+				${until.leftCol} {
+					grid-template-columns: 1fr 300px;
+					grid-template-areas:
+						'title         right-column'
+						'headline      right-column'
+						'standfirst    right-column'
+						'meta          right-column'
+						'instructions  right-column'
+						'body          right-column';
+				}
+
+				${until.desktop} {
+					grid-column-gap: 0px;
+					grid-template-columns: minmax(0, 1fr); /* Main content */
+					grid-template-areas:
+						'title'
+						'headline'
+						'standfirst'
+						'meta'
+						'instructions'
+						'body';
+				}
+
+				${until.tablet} {
+					grid-column-gap: 0px;
+					grid-template-columns: minmax(0, 1fr); /* Main content */
+					grid-template-areas:
+						'title'
+						'headline'
+						'standfirst'
+						'meta'
+						'instructions'
+						'body';
+				}
+			}
+		`}
+	>
+		{children}
+	</div>
+);
+
+const maxWidth = css`
+	${from.desktop} {
+		max-width: 620px;
+	}
+`;
+
+const stretchLines = css`
+	${until.phablet} {
+		margin-left: -20px;
+		margin-right: -20px;
+	}
+	${until.mobileLandscape} {
+		margin-left: -10px;
+		margin-right: -10px;
+	}
+`;
+
+interface CommonProps {
+	article: ArticleDeprecated;
+	format: ArticleFormat;
+	renderingTarget: RenderingTarget;
+}
+
+interface WebProps extends CommonProps {
+	NAV: NavType;
+	renderingTarget: 'Web';
+}
+
+interface AppsProps extends CommonProps {
+	renderingTarget: 'Apps';
+}
+
+export const CrosswordLayout = (props: WebProps | AppsProps) => {
+	const { article, format, renderingTarget } = props;
+	const {
+		config: { isPaidContent, host, hasSurveyAd },
+	} = article;
+
+	const isApps = renderingTarget === 'Apps';
```

Comment:
Do we need to handle the `isApps` case? My understanding is that this is layout will be web-only for the time being.

Reply:
I thought that was the case, but wasn't entirely sure. I've removed the `isApps` | `isWeb` logic for now and any app-specific components and layout.